runtime/src/accounts_db.rs

```diff
@@ -2996,27 +2996,31 @@ impl AccountsDB {
         self.accounts_cache.report_size();
     }
 
-    // `force_flush` flushes all the cached roots `<= max_clean_root`. It also then
+    // `force_flush` flushes all the cached roots `<= max_flush_root`. It also then
     // flushes:
     // 1) Any remaining roots if there are > MAX_CACHE_SLOTS remaining slots in the cache,
     // 2) It there are still > MAX_CACHE_SLOTS remaining slots in the cache, the excess
     // unrooted slots
-    pub fn flush_accounts_cache(&self, force_flush: bool, max_clean_root: Option<Slot>) {
+    pub fn flush_accounts_cache(&self, force_flush: bool, max_flush_root: Option<Slot>) {
```

Comment:
how about renaming `max_flush_root` *argument* to `requested_flush_root` across series of callgraph while keeping `AccountsCache::max_flush_root` *field*?

That's because this variable is more like similar to `SnapshotRequest` and I want to differentiate argument `max_flush_root` and `AccountsCache::max_flush_root`.

Or any other name is fine as long as it indicates we must do something up to the `Slot` regarding flushing.

I know `max_flush_root` is the first one to pop up in mind because of the already existing field name... But, engineer’s job is to give proper names to differentiate hard-to-understand workings of code. ;)

Reply:
agreed, updated!